定时任务, 遇到task无法被识别, 报错KeyError时, 例如

```python
# tasks.py
@app.task(name="你好")
def foo():
    pass

# settings.py
app.conf.beat_schedule = {
    "task_foo": {
        "task": "foo",
		...
    }
```

注意: 这里的name对应task的名字, 所以需要将setting文件修改如下

```python
app.conf.beat_schedule = {
    "task_foo": {
        "task": "你好",
		...
    }
```

